vertical-pod-autoscaler/pkg/admission-controller/certs.go

```diff
@@ -31,19 +31,14 @@ type certsConfig struct {
 }
 
 func readFile(filePath string) []byte {
-	file, err := os.Open(filePath)
+	res, err := ioutil.ReadFile(filePath)
 	if err != nil {
 		klog.Error(err)
```

Comment:
Would you mind adding some more context to this error log? sth like ("Error reading certificate file: %v", err)